crates/storage/src/transactional.rs

```diff
@@ -249,6 +283,31 @@ where
             self.storage.get(key, column)
         }
     }
+
+    fn read(
```

Comment:
I'm confused. Did we not have implementations for these methods before on `InMemoryTransaction`?

Reply:
We had a default implementation that was calling a `get` method which is less optimal in the case of RocksDB.

<img width="944" alt="image" src="https://github.com/FuelLabs/fuel-core/assets/18346821/73c8bee3-ecc9-4263-98d6-a3c6ac1bee45">